ScienceJournal/Sensors/MagnetometerSensor.swift

```diff
@@ -61,15 +62,45 @@ class MagnetometerSensor: MotionSensor {
   }
 
   override func callListenerBlocksWithData(atMilliseconds milliseconds: Int64) {
-    guard let magneticField =
-        MagnetometerSensor.motionManager.deviceMotion?.magneticField.field else { return }
+    guard let magneticField = MagnetometerSensor.motionManager.deviceMotion?.magneticField.field,
+      isValid(magneticFieldReading: magneticField)
```

Comment:
Indent to 4-in from guard rather than 2 to match style.

Reply:
Done